### PR TITLE
Installs all dependencies anticipated for use, updates README file

### PR DESCRIPTION
**Pull request by krise3**

### CODE REVIEW

**Review by harrisonywu — APPROVED** [review]
@krise3 Looks great!

**Comment by jordanstholmes** [comment]
@krise3 @harrisonywu Commit messages should be in imperative mood:
Create package-lock.json...
Add React, Redux...
Write basic CircleCI...
etc.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6.13.0
+    steps:
+      - run: echo "Placeholder step"
```
Comment (jordanstholmes) [inline .circleci/config.yml:8]: @krise3 @harrisonywu Missing newline character at end of several files. Ending a file with a newline character is a common convention as it helps certain command-line tools display those files in a way that is easier to read. Linting with AirBnB's style guide should catch this.

File: client/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+ReactDOM.render(<h1>Hello world!</h1>, document.getElementById('root'));
```
Comment (jordanstholmes) [inline client/src/index.jsx:4]: @krise3 @harrisonywu Nice job starting with the simplest component possible :D

File: package.json [hunk 3]
```diff
@@ -1,17 +1,35 @@
...
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.4.0",
+    "@babel/preset-env": "^7.4.2",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.5",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.11.2",
+    "eslint": "^5.15.3",
+    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
```
Comment (jordanstholmes) [inline package.json:30]: @krise3 @harrisonywu please use AirBnB's style guide to lint during FEC

**Pull request merged**